arrow-docs/docs/patterns/glossary/README.md

```diff
@@ -133,19 +133,19 @@ When creating an instance with the `@extension` annotation, the processor genera
 
 ```kotlin:ank:silent
 import arrow.core.Option
-import arrow.core.extensions.fx
+import arrow.core.computations.option
 
-Option.fx {
-  val (a) = Option(1)
-  val (b) = Option(a + 1)
+option.eager<Int> {
```

Comment:
do we need to specify the type here? I'm wondering since it wasn't needed before, no?

Reply:
Yes, I had to specify it now or Ank was failing. I'm not sure how it's different from before tho... 😕